src/predict/jobs/forecast/Dockerfile

```diff
@@ -11,16 +11,29 @@ RUN pip install --upgrade pip
 RUN pip install -r requirements.txt
 
 # Production
```

Comment:
Thanks @Mnoble-19 
Could we also have these comments updated to something like “production daily” and “production hourly”